io.catenax.iot_sensor_data/1.0.0/IotSensorData.ttl

```diff
@@ -0,0 +1,308 @@
+#######################################################################
+# Copyright (c) 2023 BASF SE
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 ZF Friedrichshafen AG
+#
```

Comment:
Copyright (c) 2023 Contributors to the Eclipse Foundation?

Reply:
Added. Thanks! :)